libraries/chain/steem_evaluator.cpp

```diff
@@ -1222,61 +1223,109 @@ void withdraw_vesting_evaluator::do_apply( const withdraw_vesting_operation& o )
       return;
    }
 
+#ifdef STEEM_ENABLE_SMT
+   if ( o.vesting_shares.symbol.space() == asset_symbol_type::smt_nai_space )
+   {
+      FC_ASSERT( o.vesting_shares.symbol.is_vesting(), "Vesting shares must be a vesting symbol." );
```

Comment:
Add SMT Hardfork Assertion to be sure this code cannot execute before the hardfork. It would appear safe is as an account should not be able to have an SMT token balance prior, but better to be safe. It can be removed after the hardfork.